### DISCUSSION TITLE
[Question] :Docker ?

### DISCUSSION THREAD

**Discussion by jhay06** [lima-vm/lima#1100] [Q&A] [answered]
### Description

hello ,would like to ask if this is working ?

Can i run a docker container inside lima vm environment ? 

Thanks

**Comment by AkihiroSuda** [comment, marked as answer]
Yes:
https://github.com/lima-vm/lima/blob/e4253d6490979c6ba9641d4ac0ffcdb91fe98110/examples/docker.yaml#L1-L7
